CMakeLists.txt

```diff
@@ -29,11 +31,11 @@ include(cmake/Dependencies.cmake)
 
 # ---[ Flags
 if(UNIX OR APPLE)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall -std=c++11")
```

Comment:
FYI: it's trivial to install a newer toolchain on Travis, see https://github.com/krasin/CuraEngine/commit/976b3733388c51e286e5c913b2b0d474dd9ddccd